prac_09/sort_files_2.py

```diff
@@ -0,0 +1,65 @@
+"""CP1404 - Prac 09
+Sort files 2 program
+Author: Zeb del Rosario
+Start date: 13/05/2021
+"""
+
+import os
+import shutil
+
+ROOT_DIRECTORY = os.getcwd() + "\\" + "FilesToSort"  # force cwd to be set to "FilesToSort"
+
+
+def main():
+    """Sort files v2; sort files based on user input."""
```

Comment:
Same as above. Also, use a dictionary to easily link extensions to the destination folders.